tests/run_autotests.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+NUM_FAILED=0
+
+# Determine path of the current script
+HERE=$(cd `dirname $0` && pwd)
+
+RESOURCES="${HERE}/.."
+
+# Determine paths to endless-sky executable and other relevant data
+ES_EXEC_PATH="${RESOURCES}/endless-sky"
+ES_CONFIG_PATH="${RESOURCES}/tests/config"
+ES_SAVES_PATH="${RESOURCES}/tests/config/saves"
+
+echo "***********************************************"
+echo "***         ES Autotest-runner              ***"
+echo "***********************************************"
+
+if [ ! -f "${ES_EXEC_PATH}" ] || [ ! -x "${ES_EXEC_PATH}" ]
+then
+	echo "Endless sky executable not found. Returning failure."
+	exit 1
+fi
+
+echo " Setting up environment and retrieving test-case list"
+mkdir -p "${ES_CONFIG_PATH}"
+mkdir -p "${ES_SAVES_PATH}"
+TESTS=$("${ES_EXEC_PATH}" --tests --resources "${RESOURCES}" --config "${ES_CONFIG_PATH}")
+TESTS_OK=$(echo "${TESTS}" | grep -e "ACTIVE$" | cut -d$'\t' -f1)
+TESTS_NOK=$(echo "${TESTS}" | grep -e "KNOWN FAILURE$" -e "MISSING FEATURE$" | cut -d$'\t' -f1)
+echo ""
+
+echo "Tests not to execute (known failure or missing feature):"
```

Comment:
If a test is a known failure, we should still run it. If it doesn't fail, then that is useful information.

Reply:
Every failing test currently gives a pop-up, which stops automatic test execution.
I added a TODO-item for now to make sure this doesn't get forgotten.